core/src/main/scripts/migrate_db.py

```diff
@@ -421,18 +422,35 @@ def main():
     """ main function to run mysql migration """
     parser = argparse.ArgumentParser(description='cBioPortal DB migration script')
     parser.add_argument('-y', '--suppress_confirmation', default=False, action='store_true')
-    parser.add_argument('-p', '--properties-file', type=str, required=True,
-                        help='Path to portal.properties file')
-    parser.add_argument('-s', '--sql', type=str, required=True,
-                        help='Path to official migration.sql script.')
+    parser.add_argument('-p', '--properties-file', type=str, required=False,
+                        help='Path to portal.properties file (default: locate it '
+                             'relative to the import script)')
```

Comment:
It isn't really an import script, but I don't know whether that will confuse the target audience of this usage message.